modules/hal_nordic/nrf_802154/sl_opensource/platform/nrf_802154_gpiote_zephyr.c

```diff
@@ -55,7 +55,7 @@ void nrf_802154_gpiote_init(void)
 					GPIO_PULL_UP :
 					GPIO_PULL_DOWN;
 
-#if DT_NODE_EXISTS(DT_NODELABEL(gpio1))
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(gpio1), okay)
```

Comment:
@e-rk Please take a look.
@fgervais Please submit this patch to upstream Zephyr project, https://github.com/zephyrproject-rtos/zephyr

Reply:
The change seems pretty harmless to me.
@jciupis Are the `nrf_802154_gpiote...` files still relevant? Isn't this platform file outdated at this point?

Reply:
@e-rk fair point, to my knowledge it is deprecated. However, since there is more code that would be removed together with this file I would say that it's fine to first pull in the fix from @fgervais and then proceed with deprecating and ultimately removing the platform altogether.